package.json

```diff
@@ -57,10 +57,11 @@
 		"pinst": "^3.0.0",
 		"prettier": "^2.7.1",
 		"pretty-quick": "^3.1.3",
-		"tsup": "^6.2.2",
-		"typedoc": "^0.23.10",
+		"ts-node": "^10.9.1",
```

Comment:
Before anyone asks, commitlint did a fuckywucky with one of their deps being a `.ts` config file parser requiring ts-node and it's not being loaded dynamically based on whether `ts-node` is installed or not. Just.. frigging.. great. Issues are already created. For now we need a `ts-node` devdep. Sigh.